servers/zms/src/main/java/com/yahoo/athenz/zms/utils/ZMSUtils.java

```diff
@@ -449,7 +449,7 @@ public static boolean metaValueChanged(Object domainValue, Object metaValue) {
         return metaValue != null && !metaValue.equals(domainValue);
     }
 
-    public static long configuredDueDateMillis(Integer domainDueDateDays, Integer roleDueDateDays) {
+    public static long configuredDueDateMillis(Integer serverDefaultDueDateDays, Integer domainDueDateDays, Integer roleDueDateDays) {
```

Comment:
why do we have serverDefaultDueDateDays set as Integer. It's int so we should define the function as such and it also eliminates the boxing and unnecessary null check later.